execution/executor-benchmark/src/db_generator.rs

```diff
@@ -25,6 +25,7 @@ pub fn run<V>(
     db_dir: impl AsRef<Path>,
     storage_pruner_config: PrunerConfig,
     verify_sequence_numbers: bool,
+    use_state_kv_db: bool,
```

Comment:
nit: if we have more split in the future, maybe consider passing a struct containing multiple flags so we don't need to change the function signature?

Reply:
I haven't decided yet. If that's the case I will probably pass the StroageConfig directly.